### PR TITLE
adds steps to configure cluster to be used by toolchain

### PR DESCRIPTION
**Pull request by dipak-pawar**
*Changes proposed in this PR*
This PR adds bash script to setup required things on any cluster to use that cluster by openshift.io.

With this anyone can use this script to have configuration setup required for openshift.io on any cluster

### CODE REVIEW

File: hack/setup.bash [hunk 1]
```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+oc login -u system:admin
+
+export minishift=true
```
Comment (alexeykazakov) [inline hack/setup.bash]: We will need to set up this via system env or arguments.

**Comment by stevekuznetsov** [comment]
Hey! Looks like you have misconfigured `tide`:

https://prow.svc.ci.openshift.org/tide-history?repo=fabric8-services%2Ftoolchain-operator

```
{
  "base-sha": "38a1043863527b616b460e0536485ff92f9facd8",
  "branch": "master",
  "component": "tide",
  "controller": "sync",
  "error": "Tide needs to be configured to use the 'rebase' merge method for this repo or the repo needs to allow merge commits: Merge commits are not allowed on this repository.",
  "level": "error",
  "merge-targets": [
    57
  ],
  "msg": "Merge failed.",
  "org": "fabric8-services",
  "pr": 57,
  "repo": "toolchain-operator",
  "sha": "336d3774835af55bfd94621c801c69bdf8de5747",
  "time": "2019-05-24T15:06:43Z"
}
```

**Comment by alexeykazakov** [comment]
I've enabled merge commits in the repo.

**Comment by dipak-pawar** [comment]
Thanks @stevekuznetsov for bringing it here and thank you @alexeykazakov for addressing it.

File: hack/setup.bash (hunk 1 above)
Comment (dipak-pawar) [reply hack/setup.bash]: sure we can address it later

**Pull request merged**